api/scripts/elastic_ops/mock_paragraphs/main.py

```diff
@@ -0,0 +1,80 @@
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk
+
+# import json
+# from pathlib import Path
+# from os.path import join as path_join
+
+# import random
+
+# def get_project_root() -> Path:
+#     return Path(__file__).parent
+
+
+local_es = Elasticsearch("http://localhost:9200")
+
+INDEX = "document_paragraphs"
+
+
+"""
+Script to inject paragraphs with similar document_ids
```

Comment:
I used this to create the "paragraphs with similar IDs scenario". I've been pushing all these python dev helper scripts to `dojo/api/scripts` since we have multiple operations, debugging, reproducing scenarios in the last year (even finding bad datasets when portion over es, etc) and they should help.

If anything, they're a great reference to use and copy/paste to create other quick troubleshooting scripts for the dojo stack.